xtask/src/main.rs

```diff
@@ -172,7 +172,18 @@ fn check() -> Result<()> {
 
 /// Run cargo-rdme to check if README.md is up-to-date with the library documentation
 fn check_readme() -> Result<()> {
-    run_cargo(vec!["rdme", "--workspace-project", "ratatui", "--check"])
+    let meta = MetadataCommand::new()
+        .exec()
+        .wrap_err("failed to get cargo metadata")?;
+    for package in meta.workspace_default_packages() {
```

Comment:
nit: what about making this a function? It's repeated from below.

Reply:
fa25134054c57a7a85d5deb58dde0741a8d5f121